packages/nango-yaml/lib/errors.ts

```diff
@@ -110,11 +110,11 @@ export class ParserErrorExtendsNotFound extends ParserError {
     }
 }
 
-export class ParserErrorDataSyntax extends ParserError {
+export class ParserErrorTypeSyntax extends ParserError {
     constructor(options: { value: string; path: string[] }) {
         super({
-            code: 'data_syntax_warning',
-            message: `Data type "${options.value}" contains some unsupported typescript syntax`,
+            code: 'type_syntax_error',
+            message: `Type "${options.value}" contains some unsupported typescript syntax, please use Yaml compatible syntax.`,
```

Comment:
What is `Yaml compatible syntax`? I don't think this is clear enough for the user. I see a lot of support questions stemming from this type of error message.

Reply:
I can't detail all the alternatives, there is Record, Pick, Omit, Partial, Required, Parameters, Readonly, etc. 
Happy to have a better message if you think of something 👌🏻

Reply:
I actually think this is a good time to link to the documentation with these types documented and the yaml alternative.

Reply:
There is already a section in the API references, I can point that https://docs.nango.dev/reference/integration-configuration#model-types

Reply:
Ah yes, perfect.